include/ebpf_api.h

```diff
@@ -352,7 +352,7 @@ extern "C"
      * @param[in] map_info Map to be freed.
      */
     void
-    ebpf_api_map_info_free(uint16_t map_count, _In_count_(map_count) const ebpf_map_information_t* map_info);
+    ebpf_api_map_info_free(_In_ uint16_t map_count, _In_count_(map_count) const ebpf_map_information_t* map_info);
```

Comment:
Value types should not have SAL annotation.  Rather than adding it here, it should be removed from the C file.